ambry-store/src/test/java/com.github.ambry.store/BlobStoreTest.java

```diff
@@ -578,70 +578,85 @@ public void deleteErrorCasesTest() {
   }
 
   /**
-   * Test DELETE authorization success for {@link BlobStore#delete(MessageWriteSet)}.
-   * If either accountId or containerId is -1, validation is skipped.
+   * Test DELETE with right accountId and containerId.
+   * If BlobId version is 1, accountId and containerId validation is skipped.
```

Comment:
this piece of info is an implementation detail. This function should just be checking that auth succeeds when `isAccountContainerMatch()` returns `true` and fails when it returns `false`

Reply:
fixed.